sfepy/homogenization/homogen_app.py

```diff
@@ -106,7 +112,7 @@ def update_micro_states(self):
         Update microstructures state according to the macroscopic data
         and corrector functions.
         """
-        def calculate_local_update(state, corrs, var, macro_vals):
+        def calculate_local_update(state, corrs, var, macro_vals, mul):
```

Comment:
Maybe add a default value to mul, if it is optional? (mul=1?)

Reply:
:+1: :smirk: